tests/components/prometheus/test_init.py

```diff
@@ -25,11 +54,52 @@ def test_view(prometheus_client):  # pylint: disable=redefined-outer-name
     body = yield from resp.text()
     body = body.split("\n")
 
-    assert len(body) > 3  # At least two comment lines and a metric
-    for line in body:
-        if line:
-            assert (
-                line.startswith("# ")
-                or line.startswith("process_")
-                or line.startswith("python_info")
-            )
+    assert len(body) > 3
+
+    assert "# HELP python_info Python platform information" in body
+    assert (
+        "# HELP python_gc_objects_collected_total "
```

Comment:
What message is this?

Reply:
I believe those are default ones from the prometheus client.